Felienne/hedy
# Updated to tailwind v3

**Description**
For are our front-end styling we use the Tailwind css library. A great way to quickly style the front-end and automatically generate the corresponding css files. Currently we are using version `1.9.6`. However, this version is outdated and currently version `3.0.13` is the most up-to-date. Sticking to the old version has some advantages and disadvantages:

**advantages**
- We don't break anything -> as it already works
- We use the old color scheme, colors that are not supported in versions v2 and v3
- Programmers (especially me) are already familiar with all the options of v1

**disadvantages**
- Lack off online support and documentation
- Can't use some specific new styling features
- Not supporting styling of LTR and RTL, as needed for #1679
- We have to switch someday, better today than tomorrow

In this PR we switch from our current Tailwind version to the newest one. Several configuration options are made on color schemes, line height and font-weight to make sure the new styling looks (almost) identical too the old version. Some extensive testing still has to be performed to make sure nothing is broken. For more information on the new Tailwind version take a look at: https://tailwindcss.com/blog/tailwindcss-v3

**Important sidenotes**
Some important notes that are useful for documentation and possibly future updating. Tailwind looks in `tailwind.config.js` for configurations that are different from the standard settings and overwrites these with the values in the file. In our case we overwrite the v3 color scheme with the v1 color scheme to make sure that the Hedy website stays the same color. The file `tailwind.full.js` doesn't do anything! It is a generated file of all configurations that Tailwind standard uses. It is only useful for look-up as Tailwind only looks for a file named `tailwind.config.js` for possible overwrite values. **Don't change tailwind.full.js as it doesn't do anything**.

**Fix for**
This PR fixes #1480

**How to test**
Ideally this should only be a clean-up task, but it is impossible to keep the front-end identical due to all the large changes. To test: run the branch locally and notice as little change as possible to the front-end styling.
